tests/unit/test_application.py

```diff
@@ -600,6 +600,9 @@ def test_run_error_debug(monkeypatch, mock_dispatcher, app, fake_project, error)
 _base = bases.BaseName("", "")
 _on_a_for_a = BuildInfo("p1", "a", "a", _base)
 _on_a_for_b = BuildInfo("p2", "a", "b", _base)
+_on_p3_for_p3 = BuildInfo("p3", "p3", "p3", _base)
+_on_p4_for_p3 = BuildInfo("p4", "p3", "p3", _base)
+_on_p4_for_p4 = BuildInfo("p4", "p4", "p4", _base)
```

Comment:
Are we using platform names as architectures in these build info definitions? I think it's better to use pX as platform names and single letters as architecture names otherwise this can get very confusing very fast.

Reply:
They are simulating something like:
```
platforms:
  amd64:
    build-on: amd64
    build-for: amd64
```
where the platform name matches the architecture name.